packages/frontend/src/components/Explorer/FiltersCard/FiltersCard.tsx

```diff
@@ -268,6 +268,7 @@ const FiltersCard: FC = memo(() => {
                     isEditMode={isEditMode}
                     filters={filters}
                     setFilters={setFilters}
+                    baseTable={data?.baseTable}
```

Comment:
could the base table be part of the filters provider and avoid the prop drilling?

Reply:
good point, checking!

Reply:
done here! https://github.com/lightdash/lightdash/pull/11083/commits/5dc7e9b8ea1fe523d42c126a4a63a5ca7aee3f69